framework/include/arastorage/arastorage.h

```diff
@@ -182,262 +182,270 @@ static db_output_function_t output = printf;
 ****************************************************************************/
 
 /**
-* @brief initialize database's resouces, it must be called when user arastorage
+* @brief initialize database's resources, it must be called for using arastorage
+*
+* @details @b #include <arastorage/arastorage.h>
 * @param none
-* @return On success, 1 is returned. On failure, a negative value is returned.
+* @return On success, DB_OK is returned. On failure, a negative value is returned.
 * @since Tizen RT v1.0
 */
 db_result_t db_init(void);
 
 /**
-* @brief de-initialize database's resouces, it must be called when terminate arastorage.
+* @brief de-initialize database's resouces, it must be called when terminating arastorage
+*
+* @details @b #include <arastorage/arastorage.h>
 * @param none
-* @return On success, 1 is returned. On failure, a negative value is returned.
+* @return On success, DB_OK is returned. On failure, a negative value is returned.
 * @since Tizen RT v1.0
 */
 db_result_t db_deinit(void);
 
-
 /**
-* @brief Create Component of Arastorage.
+* @brief create or remove components of arastorage
 *
```

Comment:
In my opinion instead of "components of arastorage", "relations, attributes and indexes in arastorage" would be better

Reply:
Done